File: Matrixes/1672. Richest Customer Wealth.java
package Matrixes;

class Solution{
    public int maximumWealth(int[][] accounts) {

        int maximumWealth = 0;
        int numberOfAccounts = accounts[0].length;
        for(int costumer = 0; costumer <  accounts.length; costumer ++){
            int costumerWealth = 0;
            for(int account = 0; account < numberOfAccounts; account ++)
            {
                costumerWealth += accounts[costumer][account];
            }

            if(costumerWealth > maximumWealth)
                maximumWealth = costumerWealth;
        }

        return maximumWealth;
        
    }
}

File: Matrixes/1672. Richest Customer Wealth.py
from typing import List 
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        
        richest_costumer_wealth = 0 #return variable
        
        #Loop for checking and calculating every costumer wealth.
        for costumer in accounts:
            costumer_wealth = 0 #Variable that stores the current client Wealth
            #Loop for checking every b.ank account
            for account in costumer:
                costumer_wealth += account #We add each  accounts money to the total costumer wealth
            
            #We check if this costumer is richer than the current max wealth  we have.
            if costumer_wealth > richest_costumer_wealth:
                richest_costumer_wealth =  costumer_wealth
        
        return richest_costumer_wealth
                
            
            
            
        
        
        
        